experimental/calendar/Cargo.toml

```diff
@@ -21,14 +21,3 @@ include = [
     "Cargo.toml",
     "README.md"
 ]
-
-[package.metadata.cargo-all-features]
-skip_optional_dependencies = true
-# "serde" is an intentional feature, enabling serialization of LanguageIdentifier and others:
-extra_features = ["serde"]
-
-[package.metadata.docs.rs]
-all-features = true
```

Comment:
Nit: I think we should keep

```
[package.metadata.docs.rs]
all-features = true
```

in all the Cargo.toml files so that we don't forget to add it back again. And also optionally

```
[package.metadata.cargo-all-features]
skip_optional_dependencies = true
```